Sources/AirbnbSwiftFormatTool/airbnb.swiftformat

```diff
@@ -1,3 +1,6 @@
+# ignore
```

Comment:
```suggestion
# Exclude checkout directories for common package managers
```

Reply:
I like your comment. It's clearer than mine.